shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -71,16 +79,43 @@ public void run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     AlluxioURI srcPath = new AlluxioURI(args[0]);
     AlluxioURI dstPath = new AlluxioURI(args[1]);
-    List<AlluxioURI> srcPaths = AlluxioShellUtils.getAlluxioURIs(mFileSystem, srcPath);
-    if (srcPaths.size() == 0) {
-      throw new FileDoesNotExistException(
-          ExceptionMessage.PATH_DOES_NOT_EXIST.getMessage(srcPath.getPath()));
-    }
-
-    if (srcPath.containsWildcard()) {
-      copyWildcard(srcPaths, dstPath, cl.hasOption("R"));
+    if ((dstPath.getScheme() == null || "alluxio".equals(dstPath.getScheme()))
```

Comment:
You can define a util function in this class `IsAlluxio(Scheme)` for this `or` condition.